docs/content/comparisons/druid-vs-sql-on-hadoop.md

```diff
@@ -0,0 +1,52 @@
+
+For a direct comparison between the technologies and when to only use one or the other, the basically comes down to your product requirements and what the systems were designed to do.  
+
+Druid was designed to
+
+1. be an always on service
+1. ingest data in real-time
+1. handle slice-n-dice style ad-hoc queries
+
+SQL-on-Hadoop engines (as far as we are aware) were designed to replace Hadoop MapReduce with another, faster, query layer that is completely generic and plays well with the other ecosystem of Hadoop technologies.
+
+What does this mean?  We can talk about it in terms of four general areas
+
+1. Fault Tolerance
+1. Query Speed
+1. Data Ingestion
+1. Query Flexibility
+
+## Fault Tolerance
+
+Druid pulls segments down from [Deep Storage](../dependencies/deep-storage.html) before serving queries on top of it.  This means that for the data to exist in the Druid cluster, it must exist as a local copy on a historical node.  If deep storage becomes unavailable for any reason, new segments will not be loaded into the system, but the cluster will continue to operate exactly as it was when the backing store disappeared. 
+
+SQL-on-Hadoop engines, on the other hand, pull their data from an external store such as HDFS in response to a query.  This has implications for the operation of queries if you need to take HDFS down for a bit (say a software upgrade).  It's possible that data that has been cached in the nodes is still available when the backing file system goes down, but we're not entirely sure of this.
+
+This is just one example, but Druid was built to continue operating in the face of failures of any one of its various pieces.  The [Design](../design/design.html) describes these design decisions from the Druid side in more detail.
+
+## Query Speed
+
+Druid takes control of the data given to it, storing it in a custom column-oriented fashion, compressing it and adding indexing structures.  All of which add to the speed at which queries can be processed.  The column orientation means that we only look at the data that a query asks for in order to compute the answer.  Compression increases the data storage capacity of RAM and allows us to fit more data into quickly accessible RAM.  Indexing structures mean that as you add boolean filters to your queries, we do less processing and you get your result faster, whereas a lot of processing engines do *more* processing when filters are added.
+
+SQL-on-Hadoop can basically be thought of as daemon caching layers on top of HDFS.  They are processes that stay on even if there is no query running (eliminating the JVM startup costs from Hadoop MapReduce) and they have facilities to cache data locally so that it can be accessed and updated quicker.  But, we do not believe they go beyond caching capabilities to actually speed up queries.  So, at the end of the day, they don't change the paradigm from a brute-force, scan-everything query processing paradigm.
```

Comment:
I don't think this is so accurate either. Most of the code in SQL-on-Hadoop is around query planning and execution, not dataset caching (HDFS already supports dataset caching internally).